ha/core/controllers/pcs/node_controller.py

```diff
@@ -114,7 +119,29 @@ def start(self, nodeid: str) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        _res = self.nodes_status([nodeid])
+        _node_status = _res.get(nodeid)
+        if _node_status.lower() == NODE_STATUSES.ONLINE.value.lower():
+            return {"status": "Succeeded", "msg": f"Node {nodeid}, is already in Online status"}
+        elif _node_status.lower() == NODE_STATUSES.STANDBY_WITH_RESOURCES_RUNNING.value.lower():
+            return {"status": "Succeeded", "msg": f"Node {nodeid}, is going in standby mode, "
```

Comment:
status should be "InProgress" in this case.

Reply:
Done